docker-compose.yml

```diff
@@ -38,6 +38,7 @@ services:
     image: ${MONTAGU_REGISTRY}/montagu-db:${MONTAGU_DB_VERSION}
     restart: always
     logging: *log-journald
+    command: ${MONTAGU_DB_CONF:-/etc/montagu/postgresql.conf}
```

Comment:
Is there a reason not to _always_ provide the configuration here? So for non-production environments pass through `/etc/montagu/postgresql.conf` in `compose.py` (around line 57/58)

Reply:
You're right. I had a vague idea about adding defaults to the compose file so we could use it outside of the deploy tool. But that's clearly not practical without changing all the other env vars, and quite possibly not desirable either.